src/api/routes/user.ts

```diff
@@ -0,0 +1,50 @@
+import {
+    Router,
+} from "express";
+import { Container } from "typedi";
+
+import validateUserDTO from "../middlewares/validateUserDTO";
+import UserService from "../../services/UserService";
+
+const userRouter = Router();
+const userService = Container.get(UserService);
+
+userRouter.post("/", validateUserDTO(), async (req, res) => {
```

Comment:
looks like you've missed endpoint that returns auto suggested users

Reply:
Added.